tests/containers/privileged_mode.pm

```diff
@@ -25,6 +26,7 @@ sub run {
     reset_container_network_if_needed($runtime);
 
     my $image = "registry.suse.com/bci/bci-base:latest";
+    script_retry("$runtime pull $image", delay => 60, retry => 3);
```

Comment:
```suggestion
    script_retry("$runtime pull $image", timeout => 300, delay => 120, retry => 3);
```
The default timeout for pulling is 30 seconds, and that might be too low. Let's increase the default timeout.